### PR TITLE
i have not completely made changes

### PR DESCRIPTION
**Pull request by KarthikShetty5**
I have just removed the onclick equals to logout from avatar , if u want me to add toggle also then tell me , i can quickly do that

ref: #2

### CODE REVIEW

**Comment by smitjiwani** [comment]
Yes, that would be great 👍.

**Comment by KarthikShetty5** [comment]
you want the dropdown , when the avatar is clicked ?

**Comment by KarthikShetty5** [comment]
Hey can u just check what happened in the recent changes

**Comment by smitjiwani** [comment]
Yes, that is correct. Also it would be great if you could reference the issue in the pr "ref: <reference_number>" like so

**Comment by KarthikShetty5** [comment]
Sorry , I dint get you

**Comment by smitjiwani** [comment]
Below is one of my prs, it is good practice to reference the issue at the end of the description.

Screenshot 2023-06-24 115019

**Comment by smitjiwani** [comment]
> Hey can u just check what happened in the recent changes

LGTM, once css is added. it will be ready to merge. 👍

**Comment by KarthikShetty5** [comment]
Sorry , the thing is I'm new to this , means contributing to source not for the development !
you have told me to mention ref right where should i have to mention that ,

**Comment by smitjiwani** [comment]
Its alright, dw.

**Comment by KarthikShetty5** [comment]
I have added css part , if u want it to be different please tell me

File: src/Header.js [hunk 1]
```diff
@@ -15,12 +15,25 @@ import { logout } from "./features/counter/userSlice";
...
 
   const logoutOfApp = () => {
     dispatch(logout());
     auth.signOut();
   };
+
   return (
+    <>
+    <span>
+        {dropdown && <div onClick={() => { setDropDown(true) }} onMouseLeave={() => { setDropDown(false) }} className='header_drop'>
```
Comment (smitjiwani) [inline src/Header.js]: onClick not needed, it works from below (54)

File: src/Header.css [hunk 2]
```diff
@@ -46,4 +46,23 @@
 
 .header__right{
     display: flex;
+}
+.header_drop{
```
Comment (smitjiwani) [inline src/Header.css:50]: the dropdown should be right below the avatar and to the left

File: src/Header.css [hunk 3]
```diff
@@ -46,4 +46,23 @@
...
+    position: absolute;
+    right: 36;
```
Comment (smitjiwani) [inline src/Header.css]: 36px *

File: src/Header.css [hunk 4]
```diff
@@ -46,4 +46,23 @@
...
+    align-items: center;
+    text-align: center;
+    background-color: white;
+    margin-top: 16;
+    padding-left: 4;
+    width: 30;
+    z-index: 10;
+}
+
+.account:hover{
```
Comment (smitjiwani) [inline src/Header.css]: make a single class for all three options or use '''.header_drop ul li'''

**Comment by KarthikShetty5** [comment]
Hey, im not able to run your code in codespace or vscode 
Screenshot 2023-06-24 144429

**Comment by smitjiwani** [comment]
you will have to follow the steps in readme.md and create a firestore db and put the key in firebase.js

**Comment by KarthikShetty5** [comment]
I did everything but still same

**Comment by smitjiwani** [comment]
https://discord.gg/axgcuQcT join this server

**Pull request merged**